nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -0,0 +1,283 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  cfg = config.services.glitchtip;
+  pkg = cfg.package;
+  inherit (pkg.passthru) python;
+
+  manage = pkgs.writeShellScript "manage" ''
+    set -o allexport
+    ${lib.toShellVars environment}
+    ${lib.getExe pkgs.sudo} -E -u glitchtip ${lib.getExe pkg} "$@"
+  '';
+
+  gunicornArgs = builtins.concatStringsSep " " (
+    [ "--bind=${cfg.listenAddress}:${toString cfg.port}" ] ++ cfg.gunicorn.extraArgs
+  );
+
+  celeryArgs = builtins.concatStringsSep " " cfg.celery.extraArgs;
+
+  environment = lib.mapAttrs (
+    _: value:
+    if value == true then
+      "True"
+    else if value == false then
+      "False"
+    else
+      toString value
+  ) cfg.settings;
+in
+
+{
+  meta.maintainers = with lib.maintainers; [
+    defelo
+    felbinger
+  ];
+
+  options = {
+    services.glitchtip = {
+      enable = lib.mkEnableOption "Glitchtip";
+      package = lib.mkPackageOption pkgs "glitchtip" { };
+
+      user = lib.mkOption {
+        type = lib.types.str;
+        default = "glitchtip";
+        description = "User account under which glitchtip runs.";
+      };
+
+      group = lib.mkOption {
+        type = lib.types.str;
+        default = "glitchtip";
+        description = "Group under which glitchtip runs.";
+      };
+
+      listenAddress = lib.mkOption {
+        type = lib.types.str;
+        default = "localhost";
+        description = "The address to listen on.";
+      };
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 8000;
+        description = "The port to listen on.";
+      };
+
+      settings = lib.mkOption {
+        description = ''
+          Configuration of Glitchtip. See <https://glitchtip.com/documentation/install#configuration> for more information.
+        '';
+        default = { };
+        example = {
+          SECRET_KEY = "changeme";
+          GLITCHTIP_DOMAIN = "https://glitchtip.example.com";
+          DATABASE_URL = "postgres://postgres:postgres@postgres/postgres";
```

Comment:
We should provide the socket authentication as an example as that is the most common and easiest way to deploy things on NixOS